test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Add this file (test.md) to the gitignore file. It should not be present in the remote repo.

Reply:
Thanks alot. The test.md file is already in the gitignore file. I don't know why it is showing in my repo. Please Kindly advice

Reply:
I think i have rectified the problem. Thanks